kv/memdb/memdb.s

```diff
@@ -0,0 +1,12 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
No content in this file?

Reply:
Empty file to enable `go:linkname fastRand runtime.fastrand` at `memdb.go:L310`